test/views/email/CaptureEmailViewSpec.scala

```diff
@@ -50,7 +51,8 @@ class CaptureEmailViewSpec extends ViewBaseSpec {
 
       "the user already has an email address in ETMP" should {
         lazy val view: Html = injectedView(emailForm(testEmail).fill(testEmail),
-          emailNotChangedError = false, currentEmail = testEmail)(user, messages, mockConfig)
+          emailNotChangedError = false, currentEmail = testEmail, Call("POST","/vat-through-software/account/correspondence/change-email-address")
```

Comment:
Could you put these parameters on individual lines to split up the long line length